Exponetial distribution for CLT

psanker opened the pull request.

psanker committed:
Merge remote-tracking branch 'cranmer/master'

psanker committed:
Exponential distribution

after numerous bugfixes, works appropriately

cranmer commented on distributions/Dist_psa251.py:
```diff
@@ -0,0 +1,52 @@
+from base_distribution import BaseDistribution
+
+# intentional misspelling to avoid syntax errors
+lamda = 4.
```
this is dangerous b/c if any other code that uses this defines lamba they will clash. Instead, in `__init__(self):` add a line `self.lamba=4.`. And then when you want to refer to that variable, use `self.lamba`

psanker committed:
Bugfix for lamda scope declaration

Yeah, declaring a global was dangerous. Didn’t really think about the
35 other forks potentially using the same var in the stack.

psanker commented:
Bugfix in latest push with a class variable declaration
